CRVResponse/fcl/prolog_v10.fcl

```diff
@@ -0,0 +1,241 @@
+//Use with Mu2eG4/geom/crv_counters_v08.txt (e.g. CRVResponse/test/CRVResponse.fcl)
+//The CRV-Cryo-Inner modules get a scintillation yield of 0 to disable these modules.
+//These modules cannot be removed from the geometry because it would make older files incompatible with the new geometry.
+//The geometry in Mu2eG4/geom/crv_counters_v08.txt uses a CRV counter cross section of 51.30mm x 19.80mm,
+//while the v6_0 lookup tables use a cross section of 51.34mm x 19.78mm.
+//The TiO2 coating of the CRV counters has a thickness of 0.25mm so that the active volume of the lookup table counters
+//is fully covered by the Offline counters.
+
+#include "Offline/CommonMC/fcl/prolog.fcl"
+BEGIN_PROLOG
+
+    CrvSteps:
+    {
+      module_type               : CrvStepsFromStepPointMCs
+      removeNeutralParticles    : true
+      debugLevel                : 0
+      diagLevel                 : 0
+      stepPointsInstance        : "CRV"
+      stepPointsModuleLabels    : @nil
+    }
+    CrvPhotons:
+    {
+      module_type               : CrvPhotonGenerator
+      crvStepModuleLabels       : @nil
+      crvStepProcessNames       : @nil
+      CRVSectors                : ["R1","R2","R3","R4","R5","R6","L1","L2","T1","T2","T3","T4","E1","E2","U" ,"D1","D2","D3","D4","C1","C2","C3","C4"] //used only to match the vector entries below
+      reflectors                : [  0 ,  0 , -1 ,  0 ,  0 ,  0 ,  0 ,  0 ,  1 ,  1 ,  0 ,  0 , -1 , -1 , 1  ,  0 ,  1 , -1 ,  0 ,  1 , -1 , -1 , -1 ]
+      lookupTableFileNames      : ["CRVConditions/v6_0/LookupTable_4550_0",  //R1
```

Comment:
These arrays look like they need to be synchronized to function.  In the next PR please make these into a nested table, see https://mu2e-docdb.fnal.gov/cgi-bin/sso/RetrieveFile?docid=5577&filename=ParameterSetValidation.pdf&version=1 for documentation and https://github.com/Mu2e/Offline/blob/e1a84b92c00b401039ef4cc161110458f0f17cb9/Mu2eKinKal/inc/KKFitSettings.hh#L34 for an example